audioencoder.flac/addon.xml.in

```diff
@@ -10,7 +10,7 @@
     extension=".flac"
     library_@PLATFORM@="@LIBRARY_FILENAME@"/>
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">FLAC Audio Encoder</summary>
+    <summary lang="en">FLAC (Free Lossless Audio Codec) Encoder</summary>
```

Comment:
Is better to have summary different as addon name :smile: 
![Screenshot_20190920_203408](https://user-images.githubusercontent.com/6879739/65350501-29b1f400-dbe6-11e9-9b1a-ece2e5785d48.png)

Otherwise, look a little strange if the same thing is there twice. Will soon bring Kodi requests for AAC and WMA encoder.

Also found during test some issues for Kodi:
- Change settings for audio encoder selection away from expert
- Audio Encoder addon dialog, change Button "Run" to "Use" e.g. as on screensaver
- On FLAC files included icon not shown

Is confusing if an addon can be installed and you can not easy select it.